script/numenta_benchmark.py

```diff
@@ -0,0 +1,135 @@
+import psycopg2
+import sys
+import os
+import os.path
+import csv
+import dateutil.parser
+from datetime import datetime
+import itertools
+import re
+import json
+
+# Note: initially need to run `python scripts/create_new_detector.py --detector macrobase_ma`
```

Comment:
I'd love to have numenta working long-term. What's buggy about slf4j? I'm happy to migrate if you have a preferred alternative.